packages/core/src/sdk.ts

```diff
@@ -16,5 +16,8 @@ export function initAndBind<F extends Client, O extends Options>(clientClass: Cl
   if (options.debug === true) {
     logger.enable();
   }
-  getCurrentHub().bindClient(new clientClass(options));
+  const hub = getCurrentHub();
+  const client = new clientClass(options);
+  hub.bindClient(client);
+  client.setupIntegrations();
```

Comment:
Won't this change requiring an explicit call to `client.setupIntegrations()` break places where clients are constructed outside of `Sentry.init` (which calls this `initAndBind` here)?

Reply:
Is there any other integration that need to consume configuration from `clientOptions`?